easybuild/easyconfigs/w/Wayland/Wayland-1.20.0-GCCcore-11.3.0.eb

```
@@ -0,0 +1,58 @@
+# Author: Jasper Grimm (UoY)
+easyblock = 'Bundle'
+
+name = 'Wayland'
+version = '1.20.0'
+
+homepage = 'https://wayland.freedesktop.org/'
+description = """
+Wayland is a project to define a protocol for a compositor to talk to
+ its clients as well as a library implementation of the protocol.  The
+ compositor can be a standalone display server running on Linux kernel
+ modesetting and evdev input devices, an X application, or a wayland
+ client itself.  The clients can be traditional applications, X servers
+ (rootless or fullscreen) or other display servers.
+"""
+
+toolchain = {'name': 'GCCcore', 'version': '11.3.0'}
+
+builddependencies = [
+    ('CMake', '3.23.1'),
+    ('Meson', '0.62.1'),
+    ('Ninja', '1.10.2'),
+    ('pkgconf', '1.8.0'),
+]
```

```suggestion
    ('binutils', '2.38'),
]
```

Add `binutils` to fix the pipeline